src/client/components/DateInput/index.tsx

```diff
@@ -106,7 +106,7 @@ export const DateInput: React.FC<DateInputProps> = ({
       >
         <CalendarWrapper>
           <Dayzed
-            date={new Date()}
+            date={date}
```

Comment:
I believe that in some markets, the startDate is mandatory, the original code sets automatically that value to today. What's the idea behind removing this?

Reply:
🤔 What happened is that I've checked `dayzed` library docs and [date](https://www.npmjs.com/package/dayzed#date) prop defaults to `new Date()` and it's used to _calculate what month to display on initial render_. So by keeping it as `new Date()` was resulting on the scenario where the month displayed when I open the date picker doesn't correspond with the selected date, which I think it's a not so good UX.